src/R-Tree_Intersection_Finder_C++.cpp

```diff
@@ -137,7 +143,10 @@ libsupermesh::RTree::RTree(const int &dim, const double *positions,
   : dim(dim), visitor(nelements) {
   this->memory = StorageManager::createNewMemoryStorageManager();
   this->buffer = StorageManager::createNewRandomEvictionsBuffer(*this->memory, capacity, bWriteThrough);
-    
+  
```

Comment:
This needs reimplementing by someone who knows what was going on in the first place, or at the very least knows what they are doing

Reply:
This code prevents libspatialindex swapping to disk. It is possible to avoid using using the public interface, but it's much more verbose. There's an implementation in the external_libspatialindex branch.

Reply:
Thanks! I've merged that branch in :slightly_smiling_face: